src/components/policy/test/sql_pt_representation_test.cc

```diff
@@ -1471,7 +1471,6 @@ TEST_F(SQLPTRepresentationTest,
   table["policy_table"]["app_policies"]["device"].removeMember("steal_focus");
   table["policy_table"]["app_policies"]["device"].removeMember("groups");
```

Comment:
@Kozoriz 
I think this section has to be present since it contains several parameters saved after each PTU.